### DISCUSSION TITLE
clarification on finding total_vaccine_initiated values

### DISCUSSION THREAD

**Discussion by smcclure17** [covid-projections/can-scrapers#228] [Data Issues]
From the [the MaineCountyVaccines Trello card](https://trello.com/c/L3DPqSnK/37-mainecountyvaccines-failing-since-april-3).

The issue is that `total_vaccine_initiated` is only tracking people who have received the first dose of Pfizer or Moderna series. 

I thought the solution to this was to calculate the `total_doses_initiated` as `first dose` + `final dose`, but doing this gave the same value as the `total_doses_administered` values. (Example from Washington County below)

![image](https://user-images.githubusercontent.com/55333380/114088103-ab2dad80-9882-11eb-83e7-bc5c8838a33a.png)

- _**Is this correct? What's the correct way to find `total_vaccine_initiated` from strictly first dose and completed columns?**_ 

---
definition of the data returned from the [dashboard](https://www.maine.gov/covid19/vaccines/dashboard):

![maine](https://user-images.githubusercontent.com/55333380/114085914-fe523100-987f-11eb-81a2-24a1df5705ab.PNG)

I can provide more info if helpful

**Comment by sglyon** [comment]
I believe what they are reporting as first dose is anyone who has ever received dose one of a two shot sequence (pfeizer_1 + moderna_1)

Then what they report as final is (pfeizer_2 + moderna_2 + jj)

I believe their doses administered is pfeizer_1 + moderna_1 + pfeizer_2 + moderna_2 + jj


- - -

we want initiated to be pfeizer_1 + moderna_1 + jj

I think what we need to do is:

```
total_vaccine_completed = final_dose  # their column directly
total_vaccine_doses_administered = doses_administered # their column directly
jj_doses = doses_administered - final_dose - first_dose  # all three on RHS are from their columns
total_vaccine_initiated = first_dose  + jj_doses
```

Does that make sense to you?

**Reply by smcclure17** [reply]
I see, yes. Thanks for the help

**Reply by smcclure17** [reply]
I think that `jj_doses = total_doses_admin - final_dose - first_dose` will always equal `0` because, from the def'ns, it becomes:
```
jj = (pfeizer_1 + moderna_1 + pfeizer_2 + moderna_2 + jj) - (pfeizer_2 + moderna_2 + jj) - (pfeizer_1 + moderna_1)
```
Then because `jj_doses` are included in the `final_dose` column everything gets canceled out.

I can look to see if Maine returns individual dose by manufacturer data anywhere in their dashboard/queries.

**Reply by sglyon** [reply]
Ahh yes. Good call. 

Bummer :(

Glad you were able to find a solution by getting manufacturer specific data

**Comment by smcclure17** [comment]
I've been able to get manufacturer/dose specific by county with a query. The response is a bit of a mess so it will take some time to parse--but I should be able to have this updated/fixed over the next 1 to 2 days.
